js/src/dropdown.js

```diff
@@ -450,6 +450,12 @@ class Dropdown extends BaseComponent {
       return
     }
 
+    if (!isActive && (event.key === ARROW_UP_KEY || event.key === ARROW_DOWN_KEY)) {
+      const button = this.matches(SELECTOR_DATA_TOGGLE) ? this : SelectorEngine.prev(this, SELECTOR_DATA_TOGGLE)[0]
+      button.click()
+      return
+    }
+
```

Comment:
Now the method `dataApiKeydownHandler` has a linting warning:
_Static method 'dataApiKeydownHandler' has a complexity of 23. Maximum allowed is 20._

Reply:
@rohit2sharma95 Thank you for your suggestion, let me try to figure out the best possible solution.

Reply:
As per my findings, this linting warning can be removed if we split the function `dataApiKeydownHandler`. We can create a new static method, and pass on some functionality there. 
But should we do that ?
@rohit2sharma95 @patrickhlauke @XhmikosR

Reply:
I'd say ignore it for now and tackle it later in another PR.